donate/templates/fragments/donate_form_disclaimer_master.html

```diff
@@ -4,7 +4,7 @@
     <p>
         {% block donation_information %}
         {% blocktrans with braintree_url='https://www.braintreepayments.com/legal/braintree-privacy-policy' paypal_url='https://www.paypal.com/us/webapps/mpp/ua/privacy-full' trimmed %}
-            Your payment details will be processed by <a href="{{ braintree_url }}">Braintree</a>, a PayPal company (for credit/debit cards) or <a href="{{ paypal_url }}">PayPal</a>, and a record of your donation will be stored by Mozilla.
+            Mozilla is committed to your privacy; please read our <a href="https://www.mozilla.org/privacy/websites/">privacy policy here</a>. Your payment details will be processed by <a href="{{ braintree_url }}">Braintree</a>, a PayPal company (for credit/debit cards) or <a href="{{ paypal_url }}">PayPal</a>, and a record of your donation will be stored by Mozilla.
```

Comment:
Let’s move the URL into a variable like for the other links please, so that it can’t be changed by mistake during the translation, and we won’t have to invalidate these translations if we need to change the URL (which, per Will’s comment we will do for TB soon)